src/main/java/org/jqassistant/contrib/plugin/php/model/PHPFunctionDescriptor.java

```diff
@@ -57,8 +49,34 @@
      * Number of first line in source file
      * @return integer 
      */
-    @Property("lineNumber")
-    int getLineNumber();
-    void setLineNumber(int lineNumber);
+    @Property("firstLineNumber")
```

Comment:
Thank you for the improvement. That is way clearer than before :)

After improving the property name, the documentation does not add any value - it's redundant. Please remove it.